crates/core/c8y_api/src/http_proxy.rs

```diff
@@ -205,7 +205,9 @@ impl JwtAuthHttpProxy {
     ) -> Result<JwtAuthHttpProxy, SMCumulocityMapperError> {
         let c8y_host = tedge_config.query_string(C8yUrlSetting)?;
         let device_id = tedge_config.query_string(DeviceIdSetting)?;
-        let http_con = reqwest::ClientBuilder::new().build()?;
+        let http_con = reqwest::ClientBuilder::new()
+            .danger_accept_invalid_certs(true)
```

Comment:
This __cannot__ be done for all and every connections! As said in the documentation, this [should be used as a __last resort__](https://docs.rs/reqwest/latest/reqwest/struct.ClientBuilder.html#method.danger_accept_invalid_certs).